test/helpers/index.js

```diff
@@ -0,0 +1,98 @@
+import uuidv4 from 'uuid/v4';
+import Debug from 'debug';
+import {MongoClient} from 'mongodb';
+import {Agenda, Job} from '../../lib';
+
+const mongoHost = process.env.MONGODB_HOST || 'localhost';
+const mongoPort = process.env.MONGODB_PORT || '27017';
+const debug = new Debug('agenda:test:helpers');
+const jobProcessor = () => {};
+
+const startAgenda = async agenda => {
+  return new Promise(resolve => {
+    agenda.on('ready', () => {
+      agenda.processEvery(500);
+      agenda.start();
+      debug('agenda started');
+      resolve();
+    });
+  });
+};
+
+const stopAgenda = async agenda => {
+  return new Promise(resolve => {
+    agenda.stop(() => {
+      debug('agenda stopped');
+      resolve();
+    });
+  });
+};
+
+const clearJobs = agenda => {
+  return new Promise(resolve => {
+    debug('jobs cleared');
+    agenda._mdb.collection('agendaJobs').remove({}, resolve);
```

Comment:
Should we also clean up indices here?

Reply:
We delete the whole database afterwards I was expecting that to trigger MongoDB's auto garbage collection for the indexes. Do we need to clean them ourselves?

Reply:
`Collection.remove()` leaves them there, `Collection.drop()` is what you'll need:

https://docs.mongodb.com/manual/reference/method/db.collection.drop/#db.collection.drop